api/src/paths/project/{projectId}/export/eml.ts

```diff
@@ -0,0 +1,88 @@
+import { RequestHandler } from 'express';
+import { Operation } from 'express-openapi';
+import xml2js from 'xml2js';
+import { getAPIUserDBConnection } from '../../../../database/db';
+import { EmlService } from '../../../../services/eml-service';
+import { getLogger } from '../../../../utils/logger';
+
+const defaultLog = getLogger('paths/project/{projectId}/export/eml');
+
+export const GET: Operation = [getProjectEml()];
+
+GET.apiDoc = {
+  description: 'Produces an Ecological Metadata Language (EML) extract for a target data package.',
+  tags: ['eml', 'dwc'],
+  parameters: [
+    {
+      in: 'path',
+      name: 'projectId',
+      schema: {
+        type: 'number',
+        minimum: 1
+      },
+      required: true
+    }
+  ],
+  responses: {
+    200: {
+      description: 'Ecological Metadata Language (EML) extract production OK',
+      content: {
+        'application/json': {
+          schema: {
+            title: 'Draft Get Response Object'
+          }
+        }
+      }
+    },
+    400: {
+      $ref: '#/components/responses/400'
+    },
+    401: {
+      $ref: '#/components/responses/401'
+    },
+    403: {
+      $ref: '#/components/responses/403'
+    },
+    500: {
+      $ref: '#/components/responses/500'
+    },
+    default: {
+      $ref: '#/components/responses/default'
+    }
+  }
+};
+
+export function getProjectEml(): RequestHandler {
+  return async (req, res) => {
+    defaultLog.debug({ label: 'getProjectEml', message: 'params', files: req.body });
+
+    const connection = getAPIUserDBConnection();
+
+    try {
+      const projectId = Number(req.params.projectId);
+
+      await connection.open();
+
+      const emlService = new EmlService({ projectId: projectId }, connection);
```

Comment:
NIT: This can be `const emlService = new EmlService({ projectId }, connection);`

Reply:
I know, I just havent decided how I feel about that style. It is slightly less friendly to read I find.